packages/prisma/migrations/20230404100155_remove_reminder_body_email_subject_from_reminder_template/migration.sql

```diff
@@ -0,0 +1,6 @@
+-- Until now reminderBody and emailSubject were ignored for the REMINDER template
+-- From now on we use the reminderBody and emailSubject also for templates
+
```

Comment:
Migration to make sure all reminderBodies and emailSubjects are empty for workflow steps with the REMINDER template. If reminderBody is empty we will use our normal predefined reminder template